Processors/Monolog/DeamonLoggerExtraWebProcessor.php

```diff
@@ -133,7 +133,7 @@ private function addUserInfo(): void
             return;
         }
 
-        if (!class_exists($this->userClass) && !interface_exists($this->userClass)) {
+        if ($this->userClass && !class_exists($this->userClass) && !interface_exists($this->userClass)) {
```

Comment:
I think that this statement should become something like the following : 
```suggestion
        if (null === $this->userClass || (!class_exists($this->userClass) && !interface_exists($this->userClass))) {
```